src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,72 @@
+import clsx from "clsx";
+import React from "react";
+import DropdownIcon from "./DropdownIcon";
+
+export type DropdownItem = {
+  label: string;
+  href?: string;
+  disabled?: boolean;
+  value: string;
+  selected?: boolean;
+};
+
+type DropdownProps = {
+  list: DropdownItem[];
+  ariaLabel: string;
+  arrowIcon: "triangles" | "chevron";
+  classNames?: string;
+  innerClassNames?: { select?: string; option?: string; arrowWrapper?: string };
+  handleOnChange?: (e: React.ChangeEvent<HTMLSelectElement>) => void;
+  placeholder?: string;
+};
+
+const Dropdown: React.FunctionComponent<DropdownProps> = ({
+  arrowIcon,
+  ariaLabel,
+  list,
+  classNames,
+  innerClassNames,
+  handleOnChange,
+  placeholder
+}) => {
+  const classes = {
+    root: clsx("dropdown", classNames),
+    select: clsx("dropdown__select", innerClassNames?.select),
+    option: clsx("dropdown__option", innerClassNames?.option),
+    arrowWrapper: clsx("dropdown__arrows", innerClassNames?.arrowWrapper)
+  };
+  return (
+    <div className={classes.root}>
+      <select
+        className={classes.select}
+        aria-label={ariaLabel}
+        onChange={handleOnChange ? (e) => handleOnChange(e) : undefined}
+      >
+        {placeholder && (
+          <option hidden selected>
+            {placeholder}
+          </option>
+        )}
```

Comment:
This is likely to fail accessibility tests because [the element does not have an accessible name but instead relies on a value to communicate purpose](https://accessibilityinsights.io/info-examples/web/select-name/).

I do not have a solution at hand at the moment but I think this will become a problem. At least we now have a generic component to solve it in.